horreum-backend/src/test/java/io/hyperfoil/tools/horreum/svc/SchemaServiceNoRestTest.java

```diff
@@ -0,0 +1,639 @@
+package io.hyperfoil.tools.horreum.svc;
+
+import io.hyperfoil.tools.horreum.api.SortDirection;
+import io.hyperfoil.tools.horreum.api.data.Access;
+import io.hyperfoil.tools.horreum.api.data.Extractor;
+import io.hyperfoil.tools.horreum.api.data.Label;
+import io.hyperfoil.tools.horreum.api.data.Schema;
+import io.hyperfoil.tools.horreum.api.data.Transformer;
+import io.hyperfoil.tools.horreum.api.services.SchemaService;
+import io.hyperfoil.tools.horreum.entity.data.LabelDAO;
+import io.hyperfoil.tools.horreum.entity.data.SchemaDAO;
+import io.hyperfoil.tools.horreum.entity.data.TransformerDAO;
+import io.hyperfoil.tools.horreum.test.HorreumTestProfile;
+import io.hyperfoil.tools.horreum.test.PostgresResource;
+import io.quarkus.security.ForbiddenException;
+import io.quarkus.security.UnauthorizedException;
+import io.quarkus.test.TestTransaction;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import io.quarkus.test.security.TestSecurity;
+import jakarta.inject.Inject;
+import org.hibernate.exception.SQLGrammarException;
+import org.junit.jupiter.api.Disabled;
+
+import java.util.List;
+
+import static io.hyperfoil.tools.horreum.svc.BaseServiceNoRestTest.DEFAULT_USER;
+import static io.hyperfoil.tools.horreum.svc.BaseServiceNoRestTest.FOO_TEAM;
+import static io.hyperfoil.tools.horreum.svc.BaseServiceNoRestTest.FOO_TESTER;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@QuarkusTest
+@QuarkusTestResource(PostgresResource.class)
+@TestProfile(HorreumTestProfile.class)
+@TestTransaction
+@TestSecurity(user = DEFAULT_USER, roles = {Roles.TESTER, Roles.VIEWER, FOO_TEAM, FOO_TESTER})
+class SchemaServiceNoRestTest extends BaseServiceNoRestTest {
```

Comment:
For the tests that expect `ServiceExceptions`, I think we should check the response code to validate that the service is sending the correct http reponse as well as the error message

Reply:
I agree, going to add those checks :+1:

Reply:
@lampajr have you added these checks?

Reply:
yes, see as an example: https://github.com/Hyperfoil/Horreum/pull/1656/files#diff-8f4fde724167cf6f29ef5a003807126facd504f1158a9cd6462c1be599242d81R116